docs: update security information and readme links (#1563)
AmadeusITGroup/otter: README.md

Also, to customize an application, metadata can be extracted from the application source code and injected into a CMS to manage dynamic configuration.

> [!TIP]

> > The full list of modules and their documentations is available in [/docs folder](./docs/README.md).

Reviewer: ```suggestion
> The full list of modules and their documentation is available in the [/docs folder](./docs/README.md).
```

> > A big picture of the technical module interactions and dependencies architecture is available in the [architecture section](./docs/core/ARCHITECTURE.md).

Reviewer: ```suggestion
> An overview of the technical module interactions and the architecture of the dependencies is available in the [architecture section](./docs/core/ARCHITECTURE.md).
```

  }
}
```

#### SSL Certificate issue (behind proxy)

> Due to proxy redirection you may face SSL certificate issue on Yarn when installing the Otter project locally.

Reviewer 2: you may add a quote of the type of error you can get to help people find this with search tools
> Request Error: self-signed certificate in certificate chain

To solve it you can provide your own certificate via 2 different ways:

> * By providing you certificate path in the Environment Variable [NODE_EXTRA_CA_CERTS](https://nodejs.org/docs/latest-v4.x/api/cli.html#cli_node_extra_ca_certs_file).

Reviewer 2: ```suggestion
* By providing your certificate path in the Environment Variable [NODE_EXTRA_CA_CERTS](https://nodejs.org/docs/latest-v4.x/api/cli.html#cli_node_extra_ca_certs_file).
```

  }
}
```

#### SSL Certificate issue (behind proxy)

> Due to proxy redirection you may face SSL certificate issue on Yarn when installing the Otter project locally:

Reviewer: ```suggestion
Due to proxy redirection you may face an SSL certificate issue on Yarn when installing the Otter project locally:
```

```
Request Error: self-signed certificate in certificate chain
```

> To solve it you can provide your own certificate via 2 different ways:

Reviewer: ```suggestion
To solve this, you can provide your own certificate in two different ways:
```

> * By providing your certificate path in the Environment Variable [NODE_EXTRA_CA_CERTS](https://nodejs.org/docs/latest-v4.x/api/cli.html#cli_node_extra_ca_certs_file).

Reviewer: ```suggestion
* Specify your certificate path in the Environment Variable [NODE_EXTRA_CA_CERTS](https://nodejs.org/docs/latest-v4.x/api/cli.html#cli_node_extra_ca_certs_file).
```

> * by providing your certificate path to the Yarn configuration [httpsCertFilePath](https://yarnpkg.com/configuration/yarnrc#httpsCertFilePath).

Reviewer: ```suggestion
* Specify your certificate path in the Yarn configuration [httpsCertFilePath](https://yarnpkg.com/configuration/yarnrc#httpsCertFilePath).
```
